benchmarks/nightly.sh

```diff
@@ -24,6 +24,12 @@ do
     R)
       REPEAT=${OPTARG:?}
       ;;
+    T)
+      # Avoid printing the token; re-enable printing later.
+      { set +x; } 2>/dev/null
```

Comment:
Does it really print the token otherwise?

Reply:
It really does :)
With `set -x` every bash command gets printed out -- `FOO=BAR` (i.e. bash variable assignment) or `export FOO=BAR` both get printed out.
For good measure I'm also stripping out the token from the logs/json files in case it ever leaks out of this script or through the Torchbench logs.

Reply:
Thanks for checking